datadog_checks_dev/datadog_checks/dev/tooling/utils.py

```diff
@@ -108,6 +112,10 @@ def get_valid_checks():
     return {path for path in os.listdir(get_root()) if file_exists(get_version_file(path))}
 
 
+def get_valid_integrations():
```

Comment:
Is this really different from `get_valid_checks` ?

Reply:
get_valid_checks includes any python module that has a version. This isn't 1:1 with what we consider an integration. 

I.e. datadog_checks_dev/base would be included in get_valid_checks, but wouldn't be right here
      amazon_eks doesn't have a version file but is indeed an integration with an in-app tile but wouldn't be returned via get_valid_checks.

Reply:
I see, thanks !